src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/RetrieveOrganisationsTest.java

```diff
@@ -189,6 +195,39 @@ void persists_and_returns_organisation_with_pagination() {
         assertThat(orgResponse3Size).isEqualTo(5);
     }
 
+    @Test
+    @SuppressWarnings("unchecked")
+    void persists_and_returns_organisation_with_pagination_since() throws InterruptedException {
+
+        String orgIdentifierResponse1 = createOrganisationRequest("PENDING");
+        String orgIdentifier2 = createAndActivateOrganisationWithGivenRequest(
+                someMinimalOrganisationRequest().status("ACTIVE").sraId(randomAlphabetic(10)).build());
+
+        TimeUnit.SECONDS.sleep(SINCE_PAUSE_SECONDS);
+        final LocalDateTime sinceValue = LocalDateTime.now();
+        final String since = sinceValue.format(DATE_TIME_FORMATTER);
+
+        String orgIdentifier3 = createOrganisationWithGivenRequest(

```

Comment:
there is a `ACTIVE` constant in `AuthorizationEnabledIntegrationTest` - which can be used

Reply:
makes sense